pom.xml

```diff
@@ -144,7 +143,6 @@
       <plugin>
         <groupId>org.codehaus.gmaven</groupId>
         <artifactId>gmaven-plugin</artifactId>
-        <version>${gmaven.version}</version>
```

Comment:
inherit from parent to fix some issues

Reply:
um, why removing the properties version definition but then actually you define it in dependencies?
It's better to keep all versions in central place I believe.

Reply:
Exactly. I am picking up a different version for `gmaven-plugin` from the parent; then keeping the version of `gmaven-runtime-1.8` you had (since the `-jenkins-3` patches only affect the plugin not the runtime). Since there is now only one use of `1.5`, I inlined the property.